From 18b1e75b7a3bb797a7d75d56fad1a8733c5cf989 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Mon, 15 May 2023 10:58:52 -0600
Subject: [PATCH] Replace bs4 with beautifulsoup4

`bs4` is just a dummy package for `beautifulsoup4`, so it is recommended to link to `beautifulsoup4` rather than `bs4`.
https://pypi.org/project/bs4/
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index bc0789e..ac6b4bd 100755
--- a/setup.py
+++ b/setup.py
@@ -27,6 +27,6 @@
     ],
     keywords='materials informatics crystal structures crystal-structure crystallography CRYSTAL ab-initio ab-initio-simulations first-principles materials-science',
     packages=['pycrystal'],
-    install_requires=['ase', 'pyparsing', 'requests', 'bs4'],
+    install_requires=['ase', 'pyparsing', 'requests', 'beautifulsoup4'],
     python_requires='>=3.5'
 )
